draft-thomson-httpbis-alt-svcb.md

```diff
@@ -536,14 +592,101 @@ ignore any frames other than the most recent.
 
 # Security Considerations {#security}
 
-TODO Lots of work to do here.  Review RFC 7838 for relevant information to copy over.
+Alternative services present servers with a way of influencing how clients
+select service endpoints.  This does not change how a service endpoint might be
+determined to be authoritative (even more so than its predecessor; see
+{{authority7838}}), but this.
```

Comment:
This sentence seems to stop mid-thought.

Reply:
```suggestion
determined to be authoritative (even more so than its predecessor; see
{{authority7838}}).
```